internal/convert.go

```diff
@@ -197,6 +197,29 @@ func (conv *Conv) GetDDL(c ddl.Config) []string {
 	return ddl
 }
 
+// GetFK returns the list of ALTER TABLE statements for adding
+// foreign key constraints.
+func (conv *Conv) GetFK(c ddl.Config) []string {
+	var tables []string
+	for t := range conv.SpSchema {
+		tables = append(tables, t)
+	}
+	sort.Strings(tables)
+	var ddl []string
+	// Append foreign key constraints to DDL.
+	// We always use alter table statements for foreign key constraints.
+	// The alternative of putting foreign key constraints in-line as part of create
+	// table statements is tricky because of table order (need to define tables
+	// before they are referenced by foreign key constraints) and the possibility
+	// of circular foreign keys definitions. We opt for simplicity.
+	for _, t := range tables {
+		for _, fk := range conv.SpSchema[t].Fks {
+			ddl = append(ddl, fk.PrintForeignKeyAlterTable(c, t))
+		}
+	}
```

Comment:
This piece of code is also duplicated in GetDDL, consider removing that duplication from there.

Reply:
Yes, agreed, we should avoid this duplication.

I think the simplest way is to follow what we did when adding support for foreign keys: we added a ForeignKeys field to ddl.Config control printing of foreign keys. Let's add a Table field as well, and use it to control table.s

Side discussion: the existing ForeignKey field in ddl.Config kind of breaks the ddl.go abstraction because we define a field in ddl.Config that isn't actually used/referenced in ddl.go. I think we need to do a bit of refactoring of ddl.go (in a separate PR) to resolve this. The simplest option is probably to move the GetDDL function from convert.go to ddl.go i.e. consolidate all DDL functionality in ddl.go. Asheesh -- want do you think? In any case, let's not worry about this for the moment, and we should go ahead and add a Table field to ddl.Config and resolve this PR.

Reply:
to avoid duplication, added Tables flag in Config now.

Reply:
Resolving this for now temporarily by introducing Tables field seems reasonable for me. Yes, GetDDL should be a function on ddl itself. Pradip, can you please add a TODO where Tables field is defined in ddl.Config to clean this up such that this doesn't fall through the cracks.

The way that I imagined this to work though initially was to call GetFkDDL from GetDDL if ForeignKey field was true in ddl.Config.

Reply:
I'll add TODO on GetDDL function to move it to ddl package in future.